### ISSUE TITLE
replace JitBit web console

### ISSUE THREAD

**Issue by benismobile** [edina/Estates-Audit#127]
as Jitbit administration console is third party software, we cannot do much about things like accessibility issues. If project progresses beyond pilot stage, we may want to build our own frontend to manage tickets.
